src/renderer/components/layout/BreadCrumb.tsx

```diff
@@ -56,10 +56,12 @@ class BreadCrumb extends React.Component<IProps> {
                             }}
                             title={item.name}
                         >
-                            {`${item.name} /`}
+                            {`${decodeURI(item.name)} /`}
```

Comment:
What about `title={item.name}` (just above this line)

More general question: why is `item.name` URL/percent-encoded in the first place? Shouldn't the `BreadCrumbItem.name` data *always* be "decoded" (when it corresponds to the terms of a search query)

Reply:
I correct it for the title.
The name for the serch is provided by the path data of the route, it's why it's percent encoded

Reply:
Yes I understand that "under the hood", a route path "transports" the search criteria information (thus the percent-encoding), but my argument is that the `BreadCrumbItem` object should not reflect the URL implementation detail (in other words: this object should contain data that is "ready to use", no need to decode the URI syntax).

If this is too much work to implement, then please let me know and we will merge the PR fix as it is (and I will file a separate issue to describe the improvement we can make to the implementation)